studio/lib/queries/i18n.ts

```diff
@@ -0,0 +1,28 @@
+import { groq } from "next-sanity";
```

Comment:
our root folder for translation is named `internationalization` let's rename this file to the same to more easily understand the connection

Reply:
Would it be possible to rename the root folder to `i18n` instead? Internationalization is a famously long word, and "i18n" is widely adopted 🙃

Reply:
hm, I suppose. It's just that Internationalization is the term when adding translations to websites or apps. 🤔  But if the two of you think i18n is sufficient enough without having to google it. Go ahead 👍